### DISCUSSION TITLE
q-btn v2 - How to customize borders?

### DISCUSSION THREAD

**Discussion by labs20** [quasarframework/quasar#9432] [General - Components / Directives / etc]
Hi. I'm trying to apply custom borders to some buttons on my component, but cant see it to work.

I've tried this css in both app.sass and in the <style> section:
```
.menu-btn
    .q-btn--outline:before
        border: 1px solid #eaeaea !important

    .q-btn--outline
        .q-btn__wrapper:before
            border: 1px solid #eaeaea !important

    .q-btn__wrapper:before
        border: 1px solid #eaeaea !important

```

Actually this is how the css is now after a dozen of tests.

Now, I have the `.menu-btn` class applied to some buttons, but the `.q-btn--outline:before` is not being overrided.

Any help, please?

Thank you.

**Comment by ontwikkelfabriek** [comment]
@labs20 

If you want better support please consider creating a codepen(very easy to do) to demonstrate your problem.
https://codepen.io/rstoenescu/pen/xxEvBLR?editors=101

**Reply by labs20** [reply]
Thank you. I tried to create one but had no idea how to add quasar beta on the pen. Thanks for the link, btw.

It turns out as simple as can be. I do not have to intercept the inner .q-btn class, just a simple `.menu-btn {border: ...` will do the job.

Thanks

**Reply by ontwikkelfabriek** [reply]
Very good!

btw here's a Quasar v2 codepen template:
https://codepen.io/rstoenescu/pen/xxEvBLR?editors=101

**Reply by hawkeye64** [reply]
If you want to look at advanced customization, peek at this pen: https://codepen.io/Hawkeye64/pen/WNNdKJV

**Reply by labs20** [reply]
very nice
